Druid/DRStackCompilationUnit.class.st

```diff
@@ -1,6 +1,6 @@
 Class {
 	#name : #DRStackCompilationUnit,
-	#superclass : #DRPrimitiveCompilationUnit,
+	#superclass : #DRCogitCompilationUnit,
```

Comment:
Why do we have StackCompilationUnit and CogitCompilationUnit? This is mixing middle and back ends